packages/synapse-react-client/src/components/DownloadCart/DownloadListTable.test.tsx

```diff
@@ -89,16 +83,14 @@ describe('DownloadListTable tests', () => {
   beforeEach(() => {
     jest.clearAllMocks()
   })
-  it('loads more available download files when inView', async () => {
+  it('loads more available download files when "show more" is clicked', async () => {
```

Comment:
This had an infinite scroll, which was inappropriate for a non-height constrained table that could be very long. So I changed to a "Show More" pattern